src/main/java/uk/gov/laa/ccms/caab/controller/ActionsAndNotificationsController.java

```diff
@@ -41,13 +54,25 @@ public class ActionsAndNotificationsController {
   public String getNotifications(
       @RequestParam(value = "page", defaultValue = "0") int page,
       @RequestParam(value = "size", defaultValue = "10") int size,
```

Comment:
Do we want users to be able to amend the page size?

Reply:
we could just hardcode 10 couldn't we?

Reply:
think we agreed to leave as is?